reports-scheduler/src/test/kotlin/com/amazon/opendistroforelasticsearch/reportsscheduler/rest/ReportInstanceIT.kt

```diff
@@ -0,0 +1,290 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ *
+ */
+
+package com.amazon.opendistroforelasticsearch.reportsscheduler.rest
+
+import com.amazon.opendistroforelasticsearch.reportsscheduler.PluginRestTestCase
+import com.amazon.opendistroforelasticsearch.reportsscheduler.ReportsSchedulerPlugin.Companion.BASE_REPORTS_URI
+import com.amazon.opendistroforelasticsearch.reportsscheduler.validateErrorResponse
+import org.elasticsearch.rest.RestRequest
+import org.elasticsearch.rest.RestStatus
+import org.junit.Assert
+
+class ReportInstanceIT : PluginRestTestCase() {
+    private fun constructReportDefinitionRequest(
```

Comment:
this looks like the helper method I added, should I move it to `TestHelpers` so we don't define it twice

Reply:
Yes please :)

Reply:
merged, you can remove this and import it